hashes.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
What's this file for and why do we need to add it to the codebase?

Reply:
It is to let angular build with hashes.json that is not really generated during dev builds.